Source/DataModels/Windows/SubWindows/ComponentWindows/WindowComponentScript.h

```diff
@@ -19,6 +24,12 @@ class WindowComponentScript : public ComponentWindow
 private:
 	void ChangeScript(ComponentScript* newScript, const char* selectedScript);
 
+	std::string DrawStringField(char* value, const std::string& name);
+	bool DrawBoolField(bool value, const std::string& name);
+	float DrawFloatField(float value, const std::string& name);
+	math::float3 DrawFloat3Field(math::float3 value, const std::string& name);
```

Comment:
Whoops, just realized that this float3 is passed by copy.
```suggestion
	math::float3 DrawFloat3Field(const math::float3& value, const std::string& name);
```
Sorry for not noticing before.